scripts/linters/pylint_extensions.py

```diff
@@ -2109,6 +2111,44 @@ def visit_call(self, node):
                     break
 
 
+class StringConcatenationChecker(checkers.BaseChecker):
+    """Custom pylint checker which prohibits use of string concatenation and
+     promotes string interpolation instead.
+    """
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'string-concatenation'
```

Comment:
`no-string-concatenation` - It's better to communicate what we are actually trying to enforce.

Reply:
Oh ok sure.

Reply:
Yes, you are correct done!!